package.json

```diff
@@ -102,17 +102,17 @@
     "script-loader": "^0.7.2",
     "style-loader": "^2.0.0",
     "ts-jest": "^26.4.1",
-    "ts-loader": "^8.0.5",
+    "ts-loader": "^8.0.6",
     "ts-node": "^9.0.0",
     "typescript": "^4.0.3",
     "url-loader": "^4.1.1",
     "wait-on": "^5.2.0",
-    "webpack": "^4.44.2",
-    "webpack-cli": "^3.3.12",
+    "webpack": "^5.1.3",
+    "webpack-cli": "^4.1.0",
     "webpack-dev-server": "^3.11.0"
```

Comment:
You should be able to uninstall this package now that we aren't using it.

Reply:
I think it is still needed. I think `webpack serve` is just a webpack-cli shortcut to running webpack-dev-server directly. But it is good idea to try removing it and see if `webpack serve` still works or not.

Reply:
I added a commit that removes it.  I just wanted at least one vote for removal before I yanked it.

Reply:
Anyway, I added it back in.  Sorry about that confusion.  I was a little too eager.  It certainly is needed.